### DISCUSSION TITLE
`$event` missing from list of Magics on main site

### DISCUSSION THREAD

**Discussion by joseph-ruskiewicz** [alpinejs/alpine#3952] [7. Bugs (archived)] [answered]
`$event` is not listed on the list of Magics on the sidebar.  It is on the [site](https://alpinejs.dev/essentials/events#accessing-the-event-object) but not explicitly mentioned.

**Comment by ekwoka** [comment]
Because it's not a magic that is available anywhere except inside an x-on.

**Reply by joseph-ruskiewicz** [reply, marked as answer]
It was pretty late and I needed access to event and I didn't even think to check the documentation on x-on.  I looked at the magic list and was like, weird, no $event.  Even if it was just a link to the documentation in x-on that would have been helpful to me.
